accelerators-home/modules/ROOT/pages/shared-assets.adoc

```diff
@@ -136,6 +136,6 @@ include::_partials/shared-components-fragment.adoc[]
 
 == See Also
 
-* xref:getting-started.adoc[Getting Started Guide]
+* xref:getting-started.adoc[Getting Started]
 * xref:common-services.adoc[Accelerator Common Services]
 * xref:index.adoc[MuleSoft Accelerators]
```

Comment:
I know this isn't part of the review, but you could take this opportunity to change this to match our guidance (MuleSoft Accelerators Overview).

Reply:
I'll add this as a fix for the work item I'm creating to tweak the "Simplify" section when I return.